### PR TITLE
docs(readme): simplify quick-start docs

### PR DESCRIPTION
**Pull request by missinglink**
regarding https://github.com/pelias/docker/issues/185 I would propose this as a simple solution to avoid the whole `/data` and `/code` thing

I tested this out and placing the data files in `./data` works fine, it's also nice because it makes archiving the installation much easier 🎉 

While I was at it I changed the line about permissions because it's wasn't super clear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,32 +31,34 @@ At least 8GB RAM is required.
...
+# note: use 'gsed' instead of 'sed' on a Mac
+mkdir ./data
 sed -i '/DATA_DIR/d' .env
-echo 'DATA_DIR=/data' >> .env
+echo 'DATA_DIR=./data' >> .env
+
+# configure docker to write files as your local user
+# see: https://github.com/pelias/docker#variable-docker_user
+# note: use 'gsed' instead of 'sed' on a Mac
+sed -i '/DOCKER_USER/d' .env
```
Comment (orangejulius) [inline README.md:60]: What would it take to write a command that works on both Mac and Linux?

File: README.md [hunk 2]
```diff
@@ -31,32 +31,34 @@ At least 8GB RAM is required.
...
+echo "DOCKER_USER=$(id -u)" >> .env
```
Comment (orangejulius) [inline README.md:61]: yeah, why didn't we do this before!? So simple and obvious

Comment (orangejulius) [reply README.md:61]: I guess the next step would be to make `DOCKER_USER` optional and have the scripts run `id -u` by default, thereby making the instructions even shorter.

I _think_ that was the idea behind old work like https://github.com/pelias/docker/pull/74, but it was a long time ago.

File: README.md (hunk 1 above)
Comment (missinglink) [reply README.md:60]: Probably a doctorate in computer wizardry from Hogwarts 😸

**Pull request merged**